src/server/generic_family_test.cc

```diff
@@ -24,6 +24,8 @@ using namespace util;
 using namespace boost;
 using absl::StrCat;
 
+ABSL_DECLARE_FLAG(bool, list_rdb_encode_v2);
+
```

Comment:
If you prefer, you could also use `SetTestFlag("list_rdb_encode_v2", "false");` and not declare this flag
if doesn't really matter in any way, so your call